### DISCUSSION TITLE
Foundry docker image for linux/arm64/v8

### DISCUSSION THREAD

**Discussion by h4shr4t3** [foundry-rs/foundry#7278] [Help]
Hi,

I'm currently starting my foundry docker container (to run anvil) with via this Dockerfile:

```
# syntax=docker/dockerfile:1.4
FROM ghcr.io/foundry-rs/foundry

WORKDIR /anvil

ENTRYPOINT anvil --port 7171 --host 0.0.0.0
```

It runs - but I get this warning when starting the container:

> The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

Is there a linux/arm64/v8 image available as well?

**Comment by owieth** [comment]
What happens when you install it like this? 

```sh
docker pull --platform linux/x86_64 ghcr.io/foundry-rs/foundry:master
```

Found out about passing the platform to the image if you pull it inside the Dockerfile:

```sh
FROM --platform=linux/x86_64 ghcr.io/foundry-rs/foundry:master
```

**Reply by wesfloyd** [reply]
Is there a repo developer we could ping to ask to add arm64 support to the [Docker image builds](https://github.com/foundry-rs/foundry/blob/master/.github/workflows/docker-publish.yml)? Until then, I might fork the PR, add an arm64 build target and submit a PR

**Reply by dghelm** [reply]
Also curious about this!

**Reply by stleon** [reply]
+1

**Comment by FatherShawn** [comment]
This is being worked on in #8039
